merge_snippets.sh

```diff
@@ -25,7 +25,7 @@ printf "\
 ## +-------------------------  >>d(' _ ')b<<  -------------------------+\r\n\
 ## |                                                                   |\r\n\
 ## |         Project: https://github.com/lennylxx/ipv6-hosts           |\r\n\
-## |         Update : `date    +"%a, %d %b %Y %T %z"`                  |\r\n\
+## |         Update : `LC_ALL="C" date    +"%a, %d %b %Y %T %z"`                  |\r\n\
```

Comment:
I wonder LC_ALL="C" && date +"%a, %d %b %Y %T %z", will be better? With the AND op.

Reply:
And, we don't need that much spaces before |\r\n\

Reply:
Wow,
this is a good idea but LC_ALL is a variable.
We need add export for LC_ALL="C",make it effective in this shell.

Reply:
maybe make the code longer XD

Reply:
How about you add this before date command, say line 16. And make sure it only affect the current session.

Reply:
And we can do it better, backup the current setting, once the script is finishing, restore the setting.

Reply:
Sure～we can export first then unset this variable